packages/calendar-time-utils/src/calendar-time.spec.js

```diff
@@ -79,3 +82,18 @@ describe('getLocalizedDateTimeFormatPattern', () => {
     });
   });
 });
+
+describe('formatDefaultTime', () => {
+  it('should format the time', () => {
+    expect(formatDefaultTime('09:00', 'en')).toBe('9:00 AM');
+    expect(formatDefaultTime('13:00', 'en')).toBe('1:00 PM');
```

Comment:
Can we add at least another locale example where the rendered time would be in 24 hours format?

Example:
```js
expect(formatDefaultTime('13:00', 'es')).toBe('13:00');
```